tests/autoyast/installation.pm

```diff
@@ -382,6 +384,9 @@ sub run {
         elsif (match_has_tag('opensuse-welcome')) {
             return;             # Popup itself is processed in opensuse_welcome module
         }
+        elsif (match_has_tag('bios-boot')) {
```

Comment:
I could add a comment, this is because after install we reach that [needle](http://waaa-amazing.suse.cz/tests/13874#step/installation#1/11) which has the bios-boot tag. For that reason I did extra VRs on test suites.

Reply:
this is the ay profile in pvm that I was worried about (not the link above which is installation), worth run VR for it I think: https://openqa.suse.de/tests/5247285

Reply:
Thanks, I had not found that needle, now using that one instead